src/cluecode/finder_data.py

```diff
@@ -35,6 +35,9 @@ def set_from_text(text):
 
 JUNK_EMAILS = set_from_text(u'''
     test@test.com
+    testuser
+    trialuser
+    sampleuser
```

Comment:
How are these three actual emails?
Did you add tests for that?

Reply:
They're not, was my bad to include the here. Perhaps better to leave it out now, and work on them specifically later.